pkgs/development/python-modules/dulwich/default.nix

```
@@ -17,15 +17,15 @@
 }:
 
 buildPythonPackage rec {
-  version = "0.21.5";
+  version = "0.21.6";
   pname = "dulwich";
   format = "setuptools";
 
   disabled = pythonOlder "3.6";
```

```suggestion
  disabled = pythonOlder "3.7";
```
https://github.com/jelmer/dulwich/blob/daf70786f929e3fd4995ad5d56bd93b437c20e5e/pyproject.toml#L26